test/service-data-store.test.js

```diff
@@ -24,7 +29,8 @@ describe('ListingIpfsStore load', () => {
   })
 
   it(`Should load a valid object`, async () => {
-    mockIpfsService.loadObjFromFile = sinon.stub().resolves(goodListing)
+    console.log('good listing', Object.assign({}, goodListing))
+    mockIpfsService.loadObjFromFile = sinon.stub().resolves(Object.assign({}, goodListing))
```

Comment:
That is so weird ! Why do we need to clone this object ???

Reply:
I am not sure why yet - very curious though